File: ruby/structure-in-rte/GetArticle.rb
# DocSection: structure_in_rte_retrieve_article
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client.item('coffee_beverages_explained').execute do |response|
  item = response.item
  text = item.get_string('body')
  puts text
end
# EndDocSection

File: js/secure-public-access/GetArticle.js
// DocSection: securing_public_access_get_article
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kontent-ai/delivery-sdk');

const deliveryClient = KontentDelivery.createDeliveryClient({
  projectId: '<YOUR_PROJECT_ID>',
  defaultQueryConfig: {
    useSecuredMode: true, // Queries the Delivery API using secure access.
  },
  secureApiKey: '<YOUR_API_KEY>',
});

const response = await deliveryClient.item('my_article')
  .toPromise();
// EndDocSection